### ISSUE TITLE
Does not work on 1.20.2

### ISSUE THREAD

**Issue by codecrete-ross** [comp500/BorderlessMining#88]
No matter what I do, I can't get the game to run in borderless window. The option is selected in video settings. Toggling it off/on does nothing.

**Comment by LoganDark** [comment]
Yeah the game just goes into exclusive fullscreen whenever it is focused

**Comment by leviFrosty** [comment]
I can confirm I'm also having this issue.
